lib/services.cf

```diff
@@ -340,8 +281,134 @@ bundle agent standard_services(service,state)
     verbose_mode.fallback::
       "$(this.bundle): falling back to classic_services to $(state) $(service)";
 
-    systemd.service_notfound.(start|restart|reload).(inform_mode|verbose_mode)::
-        "$(this.bundle): Could not find service: $(service)";
+}
+
+bundle agent systemd_services(service,state)
```

Comment:
Why not move these to their own files? Might make it even easier to extend for different services frameworks, granted there might only be a few others to add? termux and openrc and what does osx use?

Reply:
Do you mean like `lib/services-systemd.cf`?

We don't really subdivide other parts of the stdlib, not sure if it makes sense to do for services. How does having in separate file make it easier to extend?

Also, the network protocol is a bit chatty, so try not to be super liberal about adding files to MPF.

Reply:
good reasons. I just figure smaller files are usually easier to read in one go and easier to grok. Also easier to maintain since the diffs for systemd won't affect the main framework file. But :+1: on this due to your network protocol reason.